src/ocean/util/container/cache/ExpiringCache.d

```diff
@@ -54,13 +54,22 @@ class ExpiringCache ( size_t ValueSize = 0 ) : Cache!(ValueSize, true),
 
```

Comment:
Suggest adding this to the module doc:
"
The cache can be configured to apply different lifetimes to empty and non-empty values (where a cached record with array length 0 is considered "empty"). See constructor arguments.
"